tests/Unit/AssertTrueTest.php

```diff
@@ -170,4 +170,16 @@ function myAmazingAwesomeRecursiveFunction(int $randomNumber): bool
             myAmazingAwesomeRecursiveFunction(0)
         );
     }
+
+    public function testWithDifferentTypes(): void
+    {
+        $this->assertTrue((7 == '7'));
+        $this->assertTrue(!(7 === '7'));
+        $this->assertTrue((7 === (int) '7'));
+        $this->assertTrue($this instanceof TestCase);
+        $this->assertTrue(is_string('7'));
+        $this->assertTrue(is_int(7));
+        $this->assertTrue(is_array([]));
+        $this->assertTrue(is_bool((bool) 'false' === false));
```

Comment:
`assertTrue` and other assert methods is static, then is necessary to use `self::` instead `$this->`